GiTinder.Tests/Models/SettingsTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using GiTinder.Controllers;
+using GiTinder.Data;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Xunit;
+
+namespace GiTinder.Tests.Models
+{
+    public class SettingsTest
+    {
+        [Fact]
+        public void CanCreateSettingsWithValidUserName()
+        {
+            var settings = SettingsFactory.CreateSettingsWithValidUserName();
+            Assert.True(ValidateModel(settings).Count == 0);
+        }
+
+        [Fact]
+        public void CannotCreateSettingsWithInvalidEmptyStringUserName()
+        {
+            var settings = SettingsFactory.CreateSettingWithEmptyStringUserName();
+            Assert.True(ValidateModel(settings).Count == 1);
+        }
+
+        ////////////////////this test is failing
```

Comment:
@FilipTeply then try to fix it!!!!!!!!!!!!!!

Reply:
done